containers/ddev-webserver/Dockerfile

```diff
@@ -101,7 +101,7 @@ RUN set -x && set -o pipefail && tag=$(curl -L --fail --silent "https://api.gith
 
 RUN curl -sSL --fail --output /usr/local/bin/phive "https://phar.io/releases/phive.phar" && chmod 777 /usr/local/bin/phive
 RUN set -o pipefail && curl --fail -sSL https://github.com/pantheon-systems/terminus/releases/download/$(curl -L --fail --silent "https://api.github.com/repos/pantheon-systems/terminus/releases/latest" | perl -nle'print $& while m{"tag_name": "\K.*?(?=")}g')/terminus.phar --output /usr/local/bin/terminus && chmod 777 /usr/local/bin/terminus
-RUN set -o pipefail && curl -fsSL https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
+RUN set -o pipefail && tag=$(curl -L --fail --silent "https://api.github.com/repos/platformsh/cli/releases/latest" | jq -r .tag_name) && curl --fail -sSL "https://github.com/platformsh/cli/releases/download/${tag}/platform_${tag}_linux_${TARGETPLATFORM##linux/}.tar.gz" -o /tmp/platform.tar.gz && tar -zx -C /usr/local/bin -f /tmp/platform.tar.gz platform && rm /tmp/platform.tar.gz
```

Comment:
Since this is a go app, it should be fine like this, but remind me why you prefer to do it this way.  Why not do the Debian installation? https://github.com/platformsh/cli#ubuntudebian

Reply:
I did this to prevent situations like this:

- #5620

We have a workaround with `sudo apt update || true`, but it still increases the wait time if something is wrong with some apt repos.

I checked the script from https://github.com/platformsh/cli#ubuntudebian and saw that it also installs an additional apt repo for platform.sh.